template.rb

```diff
@@ -34,9 +34,8 @@ def render_file(filename)
 end
 
 skip_active_admin = false
-skip_devise = false
-# skip_active_admin = yes?("Skip ActiveAdmin?")
-# skip_devise = yes?("Skip Devise?")
+# skip_devise = false
+skip_devise = yes?("Skip Devise?")
```

Comment:
I think we should never skip Devise either, because ActiveAdmin needs it? Can you test whether ActiveAdmin loads it as a dependency in case the main application does not? IIRC it does not load Devise automatically because it supports multiple authentication systems.

Reply:
I created a new project with this version of the template and didn't include `devise` and I can reach the admin page. It looks like [`activeadmin` loads `devise as a dependency](https://github.com/jmw686-appdev/does-activeadmin-load-devise/blob/b6bb64016061d6f6e0af42742e99fdd1c16e9a95/Gemfile.lock#L131). @raghubetina

Reply:
Okay, in that case, LGTM :shipit: